platform/security/claims/security-claims-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -281,11 +288,37 @@ private void auditRemoteConnection(URL wsdlURL) {
     return dispatch;
   }
 
-  public void setSignatureProperties(Object signatureProperties) {
+  public void setSignaturePropertiesLocation(String signaturePropertiesLocation) {
+    this.signaturePropertiesLocation =
+        PropertyResolver.resolveProperties(signaturePropertiesLocation);
+
+    try {
+      this.setSignatureProperties(
+          PropertyResolver.resolvePropertiesFromLocation(this.signaturePropertiesLocation));
+    } catch (IOException e) {
+      LOGGER.warn(
+          "Problem reading signature properties file {}.", this.signaturePropertiesLocation);
+    }
+  }
+
+  public void setEncryptionPropertiesLocation(String encryptionPropertiesLocation) {
```

Comment:
✏️ Nit: Add unit tests for `setSignaturePropertiesLocation` and `setEncryptionPropertiesLocation`.

Reply:
TLDR: lazy

Those setters call a static method and that static method is already being tested, so I don't see much value in adding complexity to the test cases. Let me know if I am mis-judging 😄